cloudformation-template.json

```diff
@@ -197,6 +211,15 @@
 								}
 							],
 							"Id" : { "Ref" : "AWS::StackName" }
+						},
+						{
+							"DomainName" : { "Fn::Join" : [ "", [
+								{ "Ref" : "UploadsS3Bucket" }, ".s3.amazonaws.com"
+							] ] },
+							"Id" : "S3-Proxy",
+							"S3OriginConfig" : {
+
```

Comment:
@joehoyle Should this be empty? Docs seem to indicate there are required parameters? (Perhaps they just fallback to defaults when done through CloudFormation?)

Reply:
Yeah this is correct, it's quite odd - cloudformation handles the magic.